src/Microsoft.Health.Dicom.SqlServer/Features/Query/SqlQueryGenerator.cs

```diff
@@ -34,6 +34,7 @@ internal class SqlQueryGenerator : QueryFilterConditionVisitor
         private const string ExtendedQueryTagDoubleTableAlias = "ctd";
         private const string ExtendedQueryTagPersonNameTableAlias = "ctpn";
         private const string ExtendedQueryTagStringTableAlias = "cts";
+        private const int ImageResourceType = 0;
```

Comment:
Should'nt this be a enum in core library?

Reply:
Yes, Added

Reply:
Changed to enum